AUTHORS

```diff
@@ -92,6 +92,7 @@ Xiaobing Jiang <s7v7nislands at gmail.com>
 Xiuming Chen <cc at cxm.cc>
 Zhenye Xie <xiezhenye at gmail.com>
 Zhixin Wen <john.wenzhixin at gmail.com>
+Xuehong Chan <chanxuehong at gmail.com>
```

Comment:
Please keep the list alphabetically sorted, i.e. add your name after Xiuming's.

Reply:
done